spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,65 @@
+require_relative '../lib/player'
+require_relative '../lib/game'
+require_relative '../lib/board'
+require_relative '../lib/modules/helper'
+
+describe Game do
+  let(:player_one) { Player.new('Fred', 'x') }
+  let(:player_two) { Player.new('Alex', 'o') }
+  let(:game) { Game.new(player_one, player_two) }
+  let(:game_class) { Game }
+
+  describe '#choose_signs' do
+    it 'returns an array with the random signs' do
+      expect(game_class.choose_signs).to be_an(Array)
+    end
```

Comment:
- [ ] Please add negative scenario tests.

**IMPORTANT!** Please go through all your tests and missing tests i.e., add negative scenario tests (`.not_to eql`) where negative tests are missing and positive scenario tests (`.to eql`) where positive tests are missing.